packages/react-expandable-table/src/utils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+import { ASC, DESC, type Data } from './types';
+
+export const getSortedData = (
+  data: Array<Data>,
+  sortOrder: typeof ASC | typeof DESC | null,
```

Comment:
Not sure if it makes sense using null, what about using a default?

Reply:
`null` is when no sorting is enabled, so the default data ordering is used.

Reply:
OK, got it. That makes sense.